packages/jupyterlab-collaborative-chat/src/index.ts

```diff
@@ -42,11 +47,25 @@ import { YChat } from './ychat';
 const FACTORY = 'Chat';
 
 const pluginIds = {
+  acRegistry: 'jupyterlab-collaborative-chat:autocompletionRegistry',
```

Comment:
Maybe we could use a full name here so it's more explicit what it refers to?

```suggestion
  autocompletionRegistry: 'jupyterlab-collaborative-chat:autocompletionRegistry',
```